lib/ask/sox.ex

```diff
@@ -23,7 +23,42 @@ defmodule Ask.Sox do
     end
   end
 
+  # Sox doesn't support AAC directly, we must first transcode to PCM using
+  # ffmpeg then transcode to MP3 using Sox. Maybe we could transcode to MP3
+  # directly from ffmpeg but I couldn't find the arguments to set encoding,
+  # channels or rate in ffmpeg.
```

Comment:
My thoughts when reading the PRs description. I guess it'd be easier to drop Sox in favour of ffmpeg? And do a single transcode.

I guess generating a mono audio with the desired bitrate/quality should be enough - I don't think we need to provide guarantees on the encoding.

Reply:
```
ffmpeg -i "input_file.mp4" -ac 1 -ar 44100 -b:a 64k output_file.mp3
```

`-ac 1` is "mono", `-ar 44100` is sample rate at 44.1khz, `-b:a 64k` is 64kbps constant bitrate.

https://superuser.com/a/253470

Reply:
Oooh, nice, I'll look into that!